docs/source/admin/upgrading.rst

```diff
@@ -5,35 +5,43 @@ Upgrading JupyterHub
 JupyterHub offers easy upgrade pathways between minor versions. This
 document describes how to do these upgrades.
 
-If you are using :ref:`a JupyterHub distribution <index/distributions>`, you
+If you use :ref:`a JupyterHub distribution <index/distributions>`, you
 should consult the distribution's documentation on how to upgrade. This
-document is useful if you have set up your own JupyterHub without using a
+patch-2
```

Comment:
These are still here

Reply:
> These are still here

I thought I had effected all the corrections necessary. Please can you point out exactly the conflict markers you'd like me to remove before it can be merged? On what lines? The texts on line number 10?

Reply:
You should b able to read the text and see what additions are intended or not. I don't think you mean to add 'patch-2' or 'main' to the body several times. You can check the [rendered diff](https://github.com/jupyterhub/jupyterhub/pull/4113/files?short_path=f842c40#diff-f842c40cb4f2f3654d2e2991985d319c1b137a54ee86da2703189896f51115de) to see what the final page looks like, and see what text additions look incorrect to you.